### PR TITLE
Feature/пσяdƨpλя validate uniquess of podcast slug

### PR DESCRIPTION
**Pull request by nerdspar**
#1248 

- Addresses uniqueness of podcast slug
- Adds test coverage

### CODE REVIEW

**Comment by just1602** [comment]
You'll probably have to pull master since the ruby and deps upgrade is already done, but it's looks good to me.

File: spec/factories/podcasts.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :podcast do
+    created_at { Time.now.utc }
```
Comment (veganstraightedge) [inline spec/factories/podcasts.rb]:
```suggestion
```

I'm not sure you need `created_at` in your factories.

Do the tests pass with this line?

File: spec/models/podcast_spec.rb [hunk 2]
```diff
@@ -54,4 +54,26 @@
...
+      invalid_podcast = build(:podcast, slug: nil)
+
+      expect(invalid_podcast).not_to be_valid
+    end
+
+    it 'validates uniqueness of slug' do
+      valid_podcast = build(:podcast, slug: 'anarchy')
+      invalid_podcast = build(:podcast, slug: 'anarchy')
+      valid_podcast.save!
+      expect(valid_podcast).to be_valid
```
Comment (veganstraightedge) [inline spec/models/podcast_spec.rb]:
```suggestion

      expect(valid_podcast).to be_valid
```


Just a little blank space for readability.

File: spec/models/podcast_spec.rb [hunk 3]
```diff
@@ -54,4 +54,26 @@
...
+
+    it 'validates lack of presence of a slug' do
+      invalid_podcast = build(:podcast, slug: nil)
+
+      expect(invalid_podcast).not_to be_valid
+    end
+
+    it 'validates uniqueness of slug' do
+      valid_podcast = build(:podcast, slug: 'anarchy')
+      invalid_podcast = build(:podcast, slug: 'anarchy')
```
Comment (veganstraightedge) [inline spec/models/podcast_spec.rb]:
```suggestion
      invalid_podcast = build(:podcast, slug: 'anarchy')
      
```

Just a little blank space for readability.

File: Gemfile.lock [hunk 4]
```diff
@@ -328,9 +321,7 @@ GEM
     sterile (1.0.14)
       nokogiri
     stringex (2.8.5)
-    stripe (4.24.0)
-      faraday (~> 0.13)
-      net-http-persistent (~> 3.0)
+    stripe (5.1.0)
     thor (0.20.3)
     thread_safe (0.3.6)
     tilt (2.0.9)
```
Comment (veganstraightedge) [inline Gemfile.lock]: Can you delete this whole file's diff from your PR, please? 
To keep your PR focused on just your changes, and to leave the gem version bumps to Dependabot.

**Comment by nerdspar** [comment]
Updated cosmetics, tests do pass without the `created_at` in the factory.

**Comment by veganstraightedge** [comment]
Thanks for working your way through this and the feedback, @nerdspar. 🎉

**Pull request merged**